### PR TITLE
feat: Added Unsplash plugin sample

### PR DESCRIPTION
**Pull request by ashikka**
This PR closes issue #46. 

### What does it do? 
This is a plugin sample for Photoshop which lets you search for Unsplash images and display the top 10 results in a grid form inside the plugin window. Then you can click on the download button and save those images to your system.

### How to run the project? 
Please refer to the [README](https://github.com/ashikka/uxp-photoshop-plugin-samples/blob/feat/unsplash-plugin-sample/unsplash-plugin-sample/README.md) in the project for more information.

### Sample Preview

Check out the sample preview [here](https://drive.google.com/file/d/16E142uYx6zQhPgACU-LbqZwSycffzVFQ/preview)

### CODE REVIEW

File: unsplash-plugin-sample/plugin/.env.sample [hunk 1]
```diff
@@ -0,0 +1 @@
+API_BASE_URL=
```
Comment (pklaschka) [inline unsplash-plugin-sample/plugin/.env.sample:1]:
```suggestion
API_BASE_URL=http://localhost:8000
```

File: unsplash-plugin-sample/plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+For `npm` users, install all dependencies using:
+
+```
+npm install
+```
+
+## Environment Variables Setup
+
+Next, you need to create a `.env` file containing all the variables. You can check the [.env.sample](./.env.sample) file for this. Enter all variables:
```
Comment (pklaschka) [inline unsplash-plugin-sample/plugin/README.md:27]: Document what `API_BASE_URL` should be

File: unsplash-plugin-sample/plugin/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+```sh
+{
+        echo 'API_BASE_URL='
```
Comment (pklaschka) [inline unsplash-plugin-sample/plugin/README.md:31]:
```suggestion
        echo 'API_BASE_URL=http://localhost:8000'
```

File: unsplash-plugin-sample/server/.env.sample [hunk 4]
```diff
@@ -0,0 +1,2 @@
+PORT=
+ACCESS_KEY=
```
Comment (pklaschka) [inline unsplash-plugin-sample/server/.env.sample:2]:
```suggestion
PORT=8000
UNSPLASH_ACCESS_KEY=
```

File: unsplash-plugin-sample/server/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+## Environment Variables Setup
+
+Next, you need to create a `.env` file containing all the variables. You can check the [.env.sample](./.env.sample) file for this. Enter all variables:
+
+```sh
+{
+        echo 'PORT'=
+        echo 'UNSPLASH_ACCESS_KEY'=
```
Comment (pklaschka) [inline unsplash-plugin-sample/server/README.md:31]:
```suggestion
        echo 'PORT=8000'
        echo 'UNSPLASH_ACCESS_KEY='
```

File: unsplash-plugin-sample/server/README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+For `npm` users, install all dependencies using:
+
+```
+npm install
+```
+
+## Environment Variables Setup
+
+Next, you need to create a `.env` file containing all the variables. You can check the [.env.sample](./.env.sample) file for this. Enter all variables:
```
Comment (pklaschka) [inline unsplash-plugin-sample/server/README.md:26]: Document what `PORT` does (i.e., something like "`PORT` specifies the port your local API server runs on")

**Review by pklaschka — CHANGES_REQUESTED** [review]
## Review: overall
- 🟢 Overall plugin structure looks awesome!

## Review: `server`

Following `README.md` instructions:

- 🟢 `npm install` ran successfully
- 🟢 Copying `.env.sample` to `.env`
- 🔴 Change Request: `.env.sample` still contains wrongly named `ACCESS_KEY`, should be `UNSPLASH_ACCESS_KEY`
- 🟡 Minor Change Request: `PORT` should have a default value (both in the `README.md` and `.env.sample`. I.e., `PORT=8000`
- 🟢 Run `npm start` => server appears to be running successfully

## Review `plugin`
Following `README.md` instructions:

- 🟢 Run `npm install` => success
- 🟢 Copy `.env.sample` to `.env`
- 🔴 Change Request: Document what the `API_BASE_URL` should be. I assume `http://localhost:8000` for the remaining review (as this is where the NodeJS server is running)
- 🟡 Minor Change Request: Include a default value for this in both `.env.sample` and `README.md`, i.e., `API_BASE_URL=http://localhost:8000`
- 🟢 Run `npm run build` => success
- 🟢 Use UDT to load the plugin in Photoshop

## Functional plugin test
The plugin runs without problems, functionality works as specified. This was what I came up with 😉 :

cheetah

## Conclusion

Overall, this looks great! Two small change requests, but after those are done, we can merge this 🎉 . Thanks again!

File: unsplash-plugin-sample/plugin/README.md (hunk 2 above)
Comment (ashikka) [reply unsplash-plugin-sample/plugin/README.md:27]: Done

File: unsplash-plugin-sample/server/README.md (hunk 5 above)
Comment (ashikka) [reply unsplash-plugin-sample/server/README.md:31]: done

File: unsplash-plugin-sample/server/README.md (hunk 6 above)
Comment (ashikka) [reply unsplash-plugin-sample/server/README.md:26]: done

**Comment by ashikka** [comment]
@pklaschka Done! Made these changes 🚀 🥳

**Review by pklaschka — APPROVED** [review]
LGTM. Thank you for your contribution 🎉 🙂

(leaving this open for another 24 hours in case @amandahuarng or @kerrishotts want to take a look and I'll merge it afterwards)

**Pull request merged**